packages/jest-runner/src/runTest.ts

```diff
@@ -226,13 +227,20 @@ async function runTestInternal(
     let result: TestResult;
 
     try {
+      if (globalConfig.v8Coverage) {
+        await runtime.collectV8Coverage();
+      }
       result = await testFramework(
         globalConfig,
         config,
         environment,
         runtime,
         path,
       );
+
+      if (globalConfig.v8Coverage) {
+        await runtime.stopCollectingV8Coverage();
```

Comment:
we should probably run it in a `finally` block

Reply:
had it that way at first, but stopped doing it since I didn't want to do the work if the test framework exploded (it's _very_ CPU intensive). I can add an `abort` or something in the `catch` though?

Reply:
moved it